scripts/tpch_plot.py

```diff
@@ -0,0 +1,158 @@
+import itertools
+import concurrent.futures
+from pathlib import Path
+from dataclasses import dataclass
+
+from raysearch import run_and_analyze
+
+import pandas as pd
+import numpy as np
+from tqdm import tqdm
+
+
+@dataclass
+class SchedSpec:
+    name: str
+    flags: list[str]
+
+    def output_dir(self, base_dir: Path) -> Path:
+        return base_dir / self.name
+        
+
+sched_specs = {
```

Comment:
We should file a ticket to refactor out code from both `scripts/tpch_plot.py` and `scripts/raysearch.py` into a separate module that these scripts use.

The specs make it easier to group common options together and can be used in future scripts.

`scripts/raysearch.py` has some helpers to run and parse results from the simulator.